### PR TITLE
ACD meeting 109 notes

### PR DESCRIPTION
**Pull request by alita-moore**

### CODE REVIEW

File: All Core Devs Meetings/Meeting 109.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+
+### Meeting Date/Time: Friday, April 2nd, 2021 14:00 UTC
+
+### Meeting Duration: 1 hour 45 min
+
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/289)
+
+### [Video of the meeting](https://youtu.be/V-Qz4UN6Z88)
+
+### Moderator: Tim Beico
```
Comment (poojaranjan) [inline All Core Devs Meetings/Meeting 109.md:12]:
```suggestion
### Moderator: Tim Beiko
```

File: All Core Devs Meetings/Meeting 109.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+| **5** | Delay discussion on the second dev net until next meeting |
+
+## Actions Required
+
+| Action Item | Description                                                                                             |
+| ----------- | ------------------------------------------------------------------------------------------------------- |
+| **1** |  Update your node before next meeting (google for ethereum foundation blog post) |
+| **2** |  Tim to upload spec on eth1 specs repo regarding 1559 breaking changes  |
+| **3** | Danny will reach back out to EIP 3074 team about user-side security audit |
+| **4** | Thomasz will review and implement EIP 3074 more deeply to provide his perspective |
```
Comment (poojaranjan) [inline All Core Devs Meetings/Meeting 109.md:35]:
```suggestion
| **4** | Tomasz will review and implement EIP 3074 more deeply to provide his perspective |
```

File: All Core Devs Meetings/Meeting 109.md [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+
+**Tim** - Yes, I agree that generally makes sense, like the time it takes to implement them and how ready they are will be a big factor. So, yeah, maybe we do have like a pretty long list. And I know a bunch of people are on the call to kind of give an updates or discuss their EIP. So maybe it's worth taking a few minutes to go over the list and then coming back at the end to see, you know, how people feel about. Yeah, the different ones and their scope open. What the are.
+
+**Micah** - Let's focus on the easy ones, yeah, maybe like a sentence.
+
+**Tim** - Ok, so I guess, yeah, I listed them on the agenda, I guess, in the order of like. How much people have signaled roughly that they'd like this to be potentially in London or shortly after, but this is like a very subjective list.
+
+**James** - Yeah, I was going to say the first question we could triage is would would this fit in like the next step and scope of work? Just like to get a flavor for how simple it is.
+
+## 3403
```
Comment (poojaranjan) [inline All Core Devs Meetings/Meeting 109.md:175]:
```suggestion
## EIP-3403
```

File: All Core Devs Meetings/Meeting 109.md [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+
+**Martin** - I wasn't really referring to attacks using large initcodes. I think we I mean, I know the worst case, the notorious case for geth and I think client developers [would be aware of] what is the notorious case for their client. I was more thinking of their existing contract, large contracts where the initcode is more than 48K, because the incode employs not only one, but one, you know, two or three contracts at the same time. And we would just destroy them for this kind of change. That's what I mean. Not that we haven't really investiate. And I think it would be prudent to do that. 
+
+## EIP 2935 Save historical block hashes in state
```
Comment (poojaranjan) [inline All Core Devs Meetings/Meeting 109.md:371]:
```suggestion
## EIP-2935: Save historical block hashes in state
```

**Review by poojaranjan — COMMENTED** [review]
Minor fixes suggested.

**Comment by timbeiko** [comment]
@alita-moore thanks! Can you also add the call to the table on the README?

**Pull request merged**